rpc_jobs/rpc_aio.yml

```diff
@@ -286,6 +286,8 @@
                 kibana_branch = "kilo"
                 env.KIBANA_SELENIUM_BRANCH="newton-14.1"
             }}
+            // set kibana branch to newton for leapfrogs, see RE-677
+            if (env.ACTION == "leapfrogupgrade" {{ env.KIBANA_SELENIUM_BRANCH="newton" }}
```

Comment:
You could remove the assignment on line 287 since this new if block here will essentially cover that case also.